test/pytest/mocktransport.py

```diff
@@ -99,6 +99,8 @@ def stop(self, event_loop: Optional[asyncio.AbstractEventLoop] = None) -> None:
             self.close_future = event_loop.create_future()
             try:
                 event_loop.run_until_complete(self.close_future)
+            except RuntimeError:
+                pass
```

Comment:
ah wait I forgot I did this lol

Reply:
it raises `RuntimeError: This event loop is already running`, I'll probably just work around it